swagger/v1/swagger.json

```diff
@@ -0,0 +1,483 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "Forem API V1",
+    "version": "v1",
+    "description": "Access Forem articles, users and other resources via API.\n        For a real-world example of Forem in action, check out [DEV](https://www.dev.to).\n        All endpoints require an 'api-key' header and a accept header.\n\n        Dates and date times, unless otherwise specified, must be in\n        the [RFC 3339](https://tools.ietf.org/html/rfc3339) format."
```

Comment:
These multiline comments look much better in yaml. Reason use that format instead? 🤔